### ISSUE TITLE
Improve color contrast of Feedback component

### ISSUE THREAD

**Issue by katydecorah** [mapbox/dr-ui#257]
`color-gray` does not have enough contrast on top of `bg-gray-faint`. 

We'll want to either:

- use `color-gray-dark`
- consider a new theme for the Feedback component. Right now it's pretty similar to a Note and I think we could adjust the styling to make it a little more distinct.
